doc/tasks.md

```diff
@@ -1,4 +1,103 @@
 Tasks
 =====
 
-Docs coming soon.
+Task objects represent a user-created tak on a file.
+
+* [Get a Task's Information](#get-a-tasks-information)
+* [Get the Tasks on a File](#get-the-tasks-on-a-file)
+* [Add a Task to a File](#add-a-task-to-a-file)
+* [Update a Task's Information](#update-a-tasks-information)
+* [Get a Task's Assignments](#get-a-tasks-assignments)
+* [Add a Task Assignment](#add-a-task-assignment)
+
+Get a Task's Information
+---------------------------
+
+Calling [`getInfo()`][get-info] on a task returns a snapshot of the task's
+info.
+
+```java
+BoxTask task = new BoxTask(api, "id");
+BoxTask.Info info = task.getInfo();
+```
+
+[get-info]: https://box.github.io/box-java-sdk/javadoc/com/box/sdk/BoxTask.html#getInfo()
+
+Get the Tasks on a File
+--------------------------
+
+You can get all of the tasks on a file by calling the
+[`getTasks()`][get-tasks] method.
+
+```java
+BoxFile file = new BoxFile(api, "id");
+List<BoxTask.Info> tasks = file.getTasks();
+```
+
+[get-tasks]: https://box.github.io/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#getTasks()
+
+Add a Task to a File
+-----------------------
+
+A task can be added to a file with the [`addTask(String)`][add-task]
+method.
+
+```java
+BoxFile file = new BoxFile(api, "id");
+Date dueAt = new Date();
+file.addTask("review", "Please review this file.", dueAt);
+```
+
+[add-task]: https://box.github.io/box-java-sdk/javadoc/com/box/sdk/BoxFile.html#addTask(java.lang.String)
+
+Update a Task's Information
+--------------------------
+
+The message of a task can be changed with the
+[`updateInfo(String)`][update-info] method.
+
+```java
+BoxTask task = new BoxTask(api, "id");
+BoxTask.Info info = task.new Info();
+info.setMessage("An edited message.");
```

Comment:
`BoxTask.Info` currently does not expose any setters, so this example wouldn't actually work. I'll expose the necessary setters in a different PR